### DISCUSSION TITLE
Shared Adress book sorting.

### DISCUSSION THREAD

**Discussion by Nawaki89** [rustdesk/rustdesk-server-pro#254] [Q&A] [answered]
Hey I'v trying to fingure out how to sort the adress books to no avail.. is there a way?  on the discord server the CHATGTP suggested to export the books to a CSV file and sort then import again.   but I cant find any export feature  :)

any tips ?

**Comment by rustdesk** [comment]
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/71636191/b15bfa9e-b2ee-46bf-a38a-c18e214732e0)

**Reply by Nawaki89** [reply]
thank you for the answer! but i was think of the adress books them self and not clients :)  

A should be on top for reference.

I will need about 45 diffrent books so if there is a possible to sort them my self or alphabetical that would be awesome for quicker access :)

![image](https://github.com/rustdesk/rustdesk-server-pro/assets/170609271/bb9d93d0-929a-4f74-8bfc-5c39c24077ae)

**Reply by rustdesk** [reply]
@21pages let's have this sorted. excluding my address book, let's add a seperator under my address book if there are shared books.

**Reply by rustdesk** [reply, marked as answer]
https://github.com/rustdesk/rustdesk/pull/8127
